JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx

```diff
@@ -189,23 +191,84 @@ const DropZone: FC = () => {
           <Typography fontSize="16px">
             <a>Click to upload</a> or drag and drop
           </Typography>
-          <Typography color="#00000099" fontSize="14px">
+          <Typography color="#00000099" fontSize="1rem">
             CSV (max. 128MB)
           </Typography>
         </div>
       </Box>
       {uploadList}
+      <Box
+        display={'flex'}
+        justifyContent="start"
+        gap={'0.5rem'}
+        alignItems="center"
+        padding={'0.5rem'}
+      >
+        <Typography fontWeight={'bold'} fontSize={'1.2rem'}>
+          TB 1
+        </Typography>
+        <TextField placeholder="TB 1" size="small" />
+      </Box>
+      <Box
+        display={'flex'}
+        justifyContent="start"
+        gap={'0.5rem'}
+        alignItems={'center '}
+        padding={'0.5rem'}
+      >
+        <Typography fontWeight={'bold'} fontSize={'1.2rem'}>
+          CB
+        </Typography>
+        <FormControlLabel
+          value="CB2"
+          control={<Checkbox />}
+          label="CB1"
+          labelPlacement="start"
+        />
+        <FormControlLabel
+          value="CB2"
+          control={<Checkbox />}
+          label="CB2"
+          labelPlacement="start"
+        />
+      </Box>
+      <Box
+        display={'flex'}
+        gap={'1.3rem'}
+        alignItems={'center'}
+        padding={'0.5rem'}
+      >
+        <Typography fontWeight={'bold'} fontSize={'1.2rem'}>
+          RB
+        </Typography>
+        <FormControl>
+          <RadioGroup
+            aria-labelledby="demo-radio-buttons-group-label"
+            defaultValue="Opt1"
+            name="radio-buttons-group"
+          >
+            <FormControlLabel value="Opt1" control={<Radio />} label="Opt1" />
```

Comment:
Out of iinterest, what the values `Opt1`, `Opt1`, `Opt3` represent?

Reply:
It's just placeholders, the new interface is implemented you can find it here.
https://github.com/jembi/JeMPI/pull/126/commits/54a1cc053c59b4cbcdceed371f5a3b7cd8944729